src/Components/CreateImageWizardV2/steps/Review/Footer.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+
+import {
+  Button,
+  Dropdown,
+  DropdownItem,
+  DropdownList,
+  MenuToggle,
+  MenuToggleElement,
+  WizardFooterWrapper,
+  useWizardContext,
+} from '@patternfly/react-core';
+
+type ReviewWizardFooter = {
+  onSubmit?: () => void;
+  onBuild?: () => void;
+  isLoading?: boolean;
+};
+
+const ReviewWizardFooter = ({
+  onBuild,
+  onSubmit,
+  isLoading,
+}: ReviewWizardFooter) => {
+  const { goToPrevStep, close } = useWizardContext();
+  const [isOpen, setIsOpen] = useState(false);
+  const SAVE = 'save';
+  const BUILD = 'build';
```

Comment:
We could gain a bit of additional type safety by using a type here instead of constants I think:
```typescript
type submitType = 'save' | 'build'
```